src/Lepiter-Snippet-Text/LeTextSnippet.class.st

```diff
@@ -493,6 +493,16 @@ LeTextSnippet >> printOn: aStream [
 		nextPutAll: ')'
 ]
 
+{ #category : #'api - editing' }
+LeTextSnippet >> promoteToPage [
```

Comment:
@seandenigris : can you see the comment?

Reply:
> Is there a specific reason having the `#promoteToPage` in `LeTextSnippet` instead of `LeSnippet`?

Yes, the title of the resulting named page is the former string contents of the text snippet (the snippet itself is discarded and its children are moved to the new page). I wasn't sure if this conversion would be generally possible. For example, how would an image snippet get converted to a named page? Maybe it could be expanded in the future, but TextSnippet seemed like a good, easy place to start.